test/scripts/common.sh

```
@@ -97,16 +120,15 @@ wait_and_validate_cluster() {
     local expected_cpu_arch="$5"
     local cluster_type="$6"
     local expected_ssh_key="${7:-}"
-
+    local expected_platform="${8:-}"
     local cluster_name="${cluster_name_prefix}-${UNIQUE_ID}"
 
     local counter=0
     while true; do
         local cluster_data=$(fetch_cluster_data "$cluster_name")
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)